tests/plugins/test_pulp.py

```diff
@@ -150,7 +150,7 @@ def prepare(check_repo_retval=0, existing_layers=[],
     return tasker, workflow
 
 
-@pytest.mark.skipif(dockpulp is None,
+@pytest.mark.skipif(dockpulp is None,  # noqa
```

Comment:
What's the diagnostic code we're waiving here? Better to be explicit if we can be, e.g. `# noqa:W811`

Reply:
Good point.

`F811 redefinition of unused 'reactor_config_map' from line 36`

I don't think stickler actually complains about this but it's noise in flake8

Reply:
I think using `noqa:F811` (with `:`) actually silences that particular diagnostic only.

Reply:
It turns out the specific silencing only works if you use just the right format in the comment. Ironic, I know.

`# noqa: F811` silences F811
`# noqa:F811` silences everything

Reply:
:open_mouth: